### ISSUE TITLE
What Happened !

### ISSUE THREAD

**Issue by metaad** [MrAlex94/Waterfox#3347]
### What happened?

While testing G6.0.8 downloaded from (https://cdn1.waterfox.net/waterfox/releases/G6.0.8/WINNT_x86_64/Waterfox%20Setup%20G6.0.8.exe) on Windows 10 x64 VM, it was found that Product Version In Add/Remove Programs changed from G6.0.7 to G6.0.8. 
In G6.0.7 when waterfox.exe installed under "C:\Program Files\Waterfox" details are checked we found , File Version changes but not ProductVersion.

![Waterfox1](https://github.com/WaterfoxCo/Waterfox/assets/106653425/65181a81-158c-4ecf-bfac-16f12aeae40d)

and in G6.0.8
![Waterfox2](https://github.com/WaterfoxCo/Waterfox/assets/106653425/2d1a44f2-abe1-45b1-b562-6cfdbb9809e0)

Going forward will the ProductVersion change in each release or it will not change and only FileVersion will upgrade?

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

G6

### What platform are you seeing the problem on?

Windows

**Comment by MrAlex94** [comment]
Product version is essentially using the "Gecko" (browser engine) version.

Depending on when I cut the release, sometimes the Gecko version hasn't changed.

For example, often time when I cut, a version number has just been updated, but there have been no other code changes. Then when I do another release, it's after code changes have been made but no version bump has occurred yet. So in this case 115.7 was just tagged for G6.0.7 and then for G6.0.8 all the bug fixes and final release was tagged, but the version bump to 115.8 hadn't occurred yet.

Basically it depends when I cut between Thursday -> Monday after a release has been finalised by Mozilla.

**Comment by metaad** [comment]
Hi Alex,

Good Day !!

Thanks for the info, we will wait for the new release.

Regards,
PavanReddy.
